src/components/Modal/index.tsx

```diff
@@ -99,7 +102,9 @@ export default function Modal({
   maxWidth = 420,
   initialFocusRef,
   children,
+  onSwipe = onDismiss,
```

Comment:
so is the idea if we dont want a modal to be dismissable by a swipe, we pass undefined for onSwipe and a valid onDismiss function?

Reply:
Yeah, you could pass `null` explicitly, but I didn't really think about that use case.

The primary use case was to disable dismissing with overlay/ESC key (onDismiss from Overlay component) but allow gesture. Theoretically, what you are describing is possible too!